examples/windows/kubernetes.json

```diff
@@ -3,7 +3,7 @@
   "properties": {
     "orchestratorProfile": {
       "orchestratorType": "Kubernetes",
-      "orchestratorRelease": "1.13"
+      "orchestratorRelease": "1.14"

```

Comment:
@PatrickLang are you okay with this change?

Reply:
yes looks great. we should probably trim down the number of examples here if they're out of date